app/config.go

```diff
@@ -296,6 +321,10 @@ func (a *App) ClientConfigWithComputed() map[string]string {
 	// by the client.
 	respCfg["NoAccounts"] = strconv.FormatBool(a.IsFirstUserAccount())
 	respCfg["MaxPostSize"] = strconv.Itoa(a.MaxPostSize())
+	installationDate, err := a.getSystemInstallDate()
+	if err == nil {
```

Comment:
nit: should we inline here as `if installationDate, err := ...; err == nil { ... }`?

Reply:
Done!